File: https-quera.org-problemset-10231/Solution.py
def app():
    res = []
    for _ in range(1, 5 + 1):
        user_input = input()
        if "HAFEZ" in user_input or "MOLANA" in user_input:
            res.append(_)
    if len(res):
        print(*res)
    else:
        print("NOT FOUND!")


if __name__ == "__main__":
    app()

File: https-quera.org-problemset-10231/Solution.java
import java.util.ArrayList;
import java.util.List;
import java.util.Scanner;

public class Solution {

    public static void main(String[] args) {
        app();
    }

    public static void app() {
        Scanner scanner = new Scanner(System.in);
        List<Integer> res = new ArrayList<>();

        for (int i = 1; i <= 5; i++) {
            String userInput = scanner.nextLine();
            if (userInput.contains("HAFEZ") || userInput.contains("MOLANA"))
                res.add(i);
        }

        if (!res.isEmpty())
            for (int index : res)
                System.out.print(index + " ");
        else
            System.out.println("NOT FOUND!");
    }
}
